lib/uplink/example_test.go

```diff
@@ -0,0 +1,286 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package uplink_test
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	"storj.io/storj/lib/uplink"
+	"storj.io/storj/pkg/storj"
+)
+
+const (
+	myAPIKey = "change-me-to-the-api-key-created-in-satellite-gui"
+
+	satellite       = "mars.tardigrade.io:7777"
+	myBucket        = "my-first-bucket"
+	myUploadPath    = "foo/bar/baz"
+	myData          = "one fish two fish red fish blue fish"
+	myEncryptionKey = "you'll never guess this"
+)
+
+// ExampleProj_CreateBucket example documentation
+func ExampleProject_CreateBucket() {
+	var encryptionKey storj.Key
+	copy(encryptionKey[:], []byte(myEncryptionKey))
+
+	apiKey, err := uplink.ParseAPIKey(myAPIKey)
+	if err != nil {
+		log.Fatalln("could not parse api key:", err)
+	}
+
+	ctx := context.Background()
+
+	// Create an Uplink object with a default config
+	upl, err := uplink.NewUplink(ctx, nil)
+	if err != nil {
+		fmt.Println("could not create new Uplink object:", err)
```

Comment:
stick with log.Fatal for all of the error handling. fmt.Print will keep letting the program run

Reply:
Changed to log.Fatal.